### ISSUE TITLE
IE 11 Compatibility Bug 

### ISSUE THREAD

**Issue by kyjus25** [imgix/drift#75]
Hi, hoping you may be able to help sort an issue out I seem to be having. The documentation says that there is support for the 3 (or 2?) latest browsers, Edge and IE11 would fall into those categories. My issue is on Internet Explorer 11. I'm not sure if the issue is also on Edge.

I'm having an error on my site when activating Drift (the issue is also on the [Drift Playground](https://imgix.github.io/drift/)). It states that there is a syntax error on line 1, column 1770 of drift.min.js. Because of this error it fails to load and says Drift is undefined. This would be the latest version of the code as I'd just updated to the latest version from GitHub in hopes of correcting the previous issue I was having. Drift would launch once, but the second time it would throw an error about getWindowRect() or something similar. I had updated hoping it would fix the issue, but now it seems the plugin won't work at all.

Everything works perfectly on Chrome and Firefox, but unfortunately my client requires IE compatibility.

Here's a screenshot of the error that happens. If not in inspector mode the page just simply does nothing when hovering over the image

[Screenshot from Drift Playground](https://prntscr.com/kbwl1u)

**Update: I just checked: the issue does not apply to Edge.**

**Comment by frederickfogerty** [comment]
Hey there @kyjus25! Thanks for the bug report, and sorry that you're having issues with IE 11. We do support IE 11, so we'll need to fix. 

I won't have time to look into this until sometime next week, so in the meantime it would help me a lot if you'd be able to create a codesandbox.io or codepen which I can use to reproduce the issue. Would that be possible?

**Comment by kyjus25** [comment]
As requested, here's a [codepen](https://codepen.io/anon/pen/rrppvx) of a very very basic setup using Drift.

I tried to confirm that the bug is effecting this pen, but I can't even seem to fully initialize codepen with my VM of Windows 7, so I'm not sure if it's my VM or Codepen themselves don't support IE11 anymore. I've got an external hard drive enclosure on the way so that I can legitimately run Windows for compatibility testing in the future, but it won't be here till Wednesday.

Let me know if you figure anything out, but at this point it _may_ just be an issue with my VM itself, and not an issue with the plugin. There's other pens of people testing IE11 issues, so clearly codepen must be able to run on it.

I'll get back to you when I get Windows set up.
